server/routes/api.py

```diff
@@ -244,6 +244,62 @@ def post_comment(author_id: int, post_id: int) -> Response:
     return Response(status=httpStatus.OK)
 
 
+@bp.route("/authors/<int:author_id>/followers", methods=["GET"])
+def get_followers(author_id: int) -> Response:
+    followers = Requests.query.filter_by(to=author_id).all()
+    return (
+        make_response(jsonify(type="followers", items=[f.json() for f in followers])),
+        httpStatus.OK,
+    )
+
+
+@bp.route("/authors/<int:author_id>/followers/<int:follower_id>", methods=["GET"])
+def is_follower(author_id: int, follower_id: int) -> Response:
+    follower = Requests.query.filter_by(to=author_id, initiated=follower_id).first()
+    if not follower:
+        return Response(status=httpStatus.NOT_FOUND)
```

Comment:
404 errors are usually used when "the server cannot find the requested resource", since you are requesting an endpoint, then it could make it look like it didn't find the endpoint. It could make it confusing for the front-end or any user because it would seem like this endpoint was never implemented. Also in the front-end, if this error is thrown, you wouldn't know if it's because the endpoint was changed or if it's just a simple response from the back-end to a not found follower. I think that just returning a 200 with a body content that contains a boolean if it's not follower would make things simpler. I personally thought that this endpoint was not implemented and it took me a while to figure out that it was simply returning a 404 response on purpose.

Reply:
Yeah, I see what you mean.

Reply:
We can essentially just have it return an empty list. I think it would make the most sense since it stays consistent with the followers object.

Reply:
Sure! that sounds good!